src/fluree/db/time_travel.cljc

```diff
@@ -1,7 +1,7 @@
 (ns fluree.db.time-travel
   (:require [clojure.core.async :as async]
+            [fluree.db.util :as util #?(:clj :refer :cljs :refer-macros) [try* catch*]]
```

Comment:
While we're doing cleanup, I think we can remove these reader conditionals, I think `:refer` works for both clj and cljs.